index.html

```diff
@@ -6,5 +6,18 @@
     <link rel="stylesheet" type="text/css" href="styles.css" />
     <title>Quiz App</title>
   </head>
-  <body></body>
+  <body>
+    <header class="header">
+      <h1 class="header__heading">Quiz App</h1>
+    </header>
+  </body>
+
+  <section class="Question Box">
+    <h2 class="question">What property flips the axes in flexbos?</h2>
+  </section>
+
+  <footer>
+    class="footer">
```

Comment:
the opening tag is slightly broken. This should do it:

```suggestion
  <footer class="footer">
```